core/src/main/scala/kafka/server/DelayedOperation.scala

```diff
@@ -297,8 +297,7 @@ final class DelayedOperationPurgatory[T <: DelayedOperation](purgatoryName: Stri
   }
 
   /*
-   * Return the watch list of the given key, note that we need to
```

Comment:
Instead of removing this line, could we update the comment to reflect the current implementation?